p5-travel-app/src/client/js/pixabayApi.js

```diff
@@ -0,0 +1,27 @@
+const pixabayKey = "16912025-2badedd82e75bb1b12192e65c";
+
+export async function getImageURL(city, country) {
+  const queryCity = `&q=${city}&image_type=photo&pretty=true&category=places`;
+  const queryCountry = `&q=${country}&image_type=photo&pretty=true&category=places`;
+  const cityEndpoint = "https://pixabay.com/api/?key=" + pixabayKey + queryCity;
+  const countryEndpoint =
+    "https://pixabay.com/api/?key=" + pixabayKey + queryCountry;
+  try {
+    let response = await fetch(cityEndpoint);
+    //console.log(response);
```

Comment:
remove

Reply:
done in [77541ac](https://github.com/leemoispace/Udacity-Front-End-WebDeveloper/pull/9/commits/77541acca892e61c7f81d34c2e31e59fbb1a7011)